app/src/main/java/com/odysee/app/utils/Utils.java

```diff
@@ -52,6 +52,13 @@ public final class Utils {
     private static final String SP_ENCRYPTION_KEY = "key";
     private static final String SP_API_SECRET_KEY = "api_secret";
 
+    public static final int CHANNEL_THUMBNAIL_WIDTH = 160;
```

Comment:
we are using the 200x200 version in the 160 area on Odysee. Can you use that, or 100x100?"

Reply:
Using 200x200 for channel profiles shows OK on my low-end tablet, so I think 200x200 is a good value for all devices. Android will resize it if needed.